plugins/inputs/mesos/mesos_test.go

```diff
@@ -366,31 +368,31 @@ func TestMesosMaster(t *testing.T) {
 		// framework offers
 		{
 			// "unknown" metric type should still contain framework_name tag
-			"master/frameworks/unknown/unknown":  masterMetrics["master/frameworks/marathon/abc-123/unknown/unknown"],
-			"master/frameworks/calls_total":      masterMetrics["master/frameworks/marathon/abc-123/calls"],
-			"master/frameworks/events_total":     masterMetrics["master/frameworks/marathon/abc-123/events"],
-			"master/frameworks/operations_total": masterMetrics["master/frameworks/marathon/abc-123/operations"],
-			"master/frameworks/subscribed_total": masterMetrics["master/frameworks/marathon/abc-123/subscribed"],
-			"master/frameworks/offers/sent":      masterMetrics["master/frameworks/marathon/abc-123/offers/sent"],
+			"master/frameworks/unknown/unknown":  masterMetrics["master/frameworks/framework+name/abc-123/unknown/unknown"],
```

Comment:
My understanding is that mesos percent-encodes, rather than url-encoding, framework names. Hence this would be `framework%20name` rather than `framework+name`

Reply:
Thanks, this made me realize that I actually want `PathUnescape()`: https://golang.org/pkg/net/url/#PathUnescape